src/test/java/hudson/plugins/performance/PerformancePipelineTest.java

```diff
@@ -0,0 +1,69 @@
+package hudson.plugins.performance;
+
+import hudson.model.Result;
+import hudson.slaves.DumbSlave;
+import org.apache.commons.io.IOUtils;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import java.net.URL;
+
+public class PerformancePipelineTest  {
+    @ClassRule
+    public static BuildWatcher buildWatcher = new BuildWatcher();
+
+    @Rule
+    public RestartableJenkinsRule story = new RestartableJenkinsRule();
+
+    @Test
+    public void bztSmokeTests() throws Exception {
+        String path = getClass().getResource("/performanceTest.yml").getPath();
+        final String bztParams =  path + ' ' + "-o modules.jmeter.plugins=[] -o services=[]";
```

Comment:
For fix tests on travis add the next options "-o modules.jmeter.version=3.1 -o modules.jmeter.path=PATH_TO_WORKSPACE" like here PerformanceTestBuildTest#testFlow#L40